JSDemos/Demos/PivotGrid/OLAPDataSource/description.md

```diff
@@ -1 +1,18 @@
-The PivotGrid supports OLAP services (MS&nbsp;SQL Server Analysis Services). This allows you to&nbsp;deliver solutions that can easily analyze millions of&nbsp;data rows within your app. In&nbsp;this demo, the PivotGrid uses **XmlaStore** to&nbsp;connect the component to&nbsp;a&nbsp;sample remote OLAP service. The Field Chooser is&nbsp;configured to&nbsp;show the search box to&nbsp;speed up&nbsp;choosing the required field.
\ No newline at end of file
+The [PivotGrid](/Documentation/Guide/UI_Components/PivotGrid) component supports OLAP services (MS SQL Server Analysis Services). This demo shows how to use a remote OLAP cube as the PivotGrid's data source.
+
+**Configure Store**
+To connect your component to an OLAP storage, use [XmlaStore](/Documentation/ApiReference/Data_Layer/XmlaStore/) as an underlying store of the PivotGrid's [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource). Specify the following properties to configure the **XmlaStore**:    
+
+- [url](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#url)        
+The OLAP server's URL.
+
+- [catalog](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#catalog)        
+The initial catalog that contains the OLAP cube.
+
+- [cube](/Documentation/ApiReference/Data_Layer/XmlaStore/Configuration/#catalog)       
+The name of the OLAP cube to use from the catalog.
+
+**Configure PivotGrid Fields**
+Assign an array of objects to the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) property. Each object in this array configures a single field. Fields can belong to different [areas](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area). Add OLAP measures to the *"data"* area dimensions to the *"column"* and *"row"* areas. You can use OLAP dimensions to group and filter measure values from the *"data"* area. This demo shows how to use the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to access the measures and specify ways to these measures in the OLAP storage. 
```

Comment:
Fields are configured in the same way regardless of whether an OLAP cube or local array is used. I think we can copy the description from the Simple Array demo.